_IBC/fxcore-cosmoshub.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "../ibc_data.schema.json",
+    "chain_1": {
+      "chain_name": "fxcore",
```

Comment:
to avoid duplicates, the chain1:chain2 ordering must be alphabetical, so cosmoshub should be chain1 and fxcore as chain2